src/components/Home.js

```diff
@@ -26,61 +26,56 @@ class Home extends Component {
       <div className="home">
         <div className="title">
           Bitcoin,
-		      <br />
+          <br />
           On
-		      <br />
+          <br />
           Ethereum
-		      <div className="subtitle">
+          <div className="subtitle">
             <div className="vertical-aligned">
-              { isMobile
-                ? <span>
-                    No games,
-                    <br />
-                    just math.
-                  </span>
-                : <span>No games, just math.</span>
-              }
-		        </div>
+              {isMobile ? (
+                <span>
+                  No games,
+                  <br />
+                  just math.
+                </span>
+              ) : (
+                <span>No games, just math.</span>
+              )}
+            </div>
           </div>
         </div>
-        {
-          noEntry
-          ? ''
-          : (
-            <div className="mint-or-redeem">
-              <a href="/deposit" onClick={this.handleClickDeposit}>
-                Deposit
-              </a>
-              <a href="/redeem" onClick={this.handleClickRedeem}>
-                Redeem
-              </a>
-            </div>
-          )
-        }
+        {noEntry ? (
+          ""
+        ) : (
+          <div className="mint-or-redeem">
+            <a href="/deposit" onClick={this.handleClickDeposit}>
+              Deposit
+            </a>
+            <a href="/redeem" onClick={this.handleClickRedeem}>
+              Redeem
+            </a>
+          </div>
+        )}
```

Comment:
We can probably lose `noEntry` too, though that might be off-topic here 😅 I think that's from when a version of the dApp was also the static tBTC site.